pkg/controller/plan/kubevirt.go

```diff
@@ -945,8 +907,7 @@ func (r *KubeVirt) DeleteObject(object client.Object, vm *plan.VMStatus, message
 
 // Delete any hook jobs that belong to a VM migration.
 func (r *KubeVirt) DeleteHookJobs(vm *plan.VMStatus) (err error) {
-	vmLabels := r.vmLabels(vm.Ref)
-	delete(vmLabels, kMigration)
```

Comment:
actually, getting back to this PR after a while, I wanted to suggest to do something `filter(vmLabels(..))` which is basically equivalent to how it was before - isn't it more readable? maybe adding another function like `vmAllButMigrationLabels`?